### PR TITLE
Update to drupal 8.7.9

### PR DESCRIPTION
**Pull request by anthony-malkoun**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-3272

### Changed

1.  Update Drupal to 8.7 and other updates as required.

### CODE REVIEW

File: dev-tools.sh [hunk 1]
```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
```
Comment (anthony-malkoun) [inline dev-tools.sh]: Do not merge. For testing only until a release of `dev-tools`

File: composer.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
         "drupal/seckit": "^1.1",
         "drupal/simple_sitemap": "^2.12",
         "drupal/smart_trim": "^1.1",
-        "drupal/stage_file_proxy": "^1.0-alpha3",
+        "drupal/stage_file_proxy": "1.0-rc2",
```
Comment (anthony-malkoun) [inline composer.json]: This version of `stage_file_proxy` is for Drupal >= 8.7.7

**Comment by vincent-gao** [comment]
Updated with develop

**Comment by vincent-gao** [comment]
another updating with develop
cc @anthony-malkoun

File: dev-tools.sh (hunk 1 above)
Comment (vincent-gao) [reply dev-tools.sh]: we will merge it. and delete it after all of client sites get updated to 8.7+
@anthony-malkoun

**Comment by vincent-gao** [comment]
```
  Problem 1
    - Installation request for dpc-sdp/tide_core @dev -> satisfiable by dpc-sdp/tide_core[dev-feature/SDPA-3272-update-drupal-8-7].
    - dpc-sdp/tide_core dev-feature/SDPA-3272-update-drupal-8-7 requires drupal/core 8.7.9 -> satisfiable by drupal/core[8.7.9] but these conflict with your requirements or minimum-stability.
```

after discussed with @anthony-malkoun, we decide to merge the branch. just ignore the failed building.
the dev branch will always be drupal 8.6 version, there is no way to fix it without merging.

**Pull request merged**